package.json

```diff
@@ -37,6 +37,10 @@
     "prettier-check": "ts-node ./scripts/prettier-check.ts",
     "prettier-fix": "prettier --write specification/**/*.json",
     "prettier": "prettier",
-    "tsc": "tsc"
+    "tsc": "tsc",
```

Comment:
Can we revert the changes in package.json and package-lock.json? These apply to all Azure services